main/getting-started/intro-zoe.md

```diff
@@ -1,358 +1,286 @@
-</td>
-</tr>
-<tr>
-<td>5</td>
-<td>The contract code processes the offers, which may request some reallocations as a result.</td>
-</tr>
-<tr>
-<td>6</td>
-<td>Zoe reallocates the assets among the offers, preserving offer safety.
-</td>
-</tr>
-<tr>
-<td>7</td>
-<td>The contract requests Zoe process the satisfied offers, resolving their associated payout promises. Offer holders either get what they paid for, get their escrowed offer amount refunded, or in some cases, get both what they paid for and a refund of any overpayment.
-</td>
-</tr>
-</tbody>
-</table>
+<<< @/snippets/test-intro-zoe.js#harden
+
+Now that we have the required pieces, we can make an offer:
+
+<<< @/snippets/test-intro-zoe.js#offer
+
+At this point, Zoe burns our invitation and confirms its validity. Zoe
+also escrows all of our payments and represents their value in amounts
+as our `current allocation` in the contract.
+
+### Obtaining a UserSeat
+
+Making an offer as a user gives back a `UserSeat`. This seat
+represents our position in the ongoing contract. We can use this seat
+to do a number of things, including:
+
+1. exit the contract
+2. get information about our position such as our current allocation
+3. get our payouts from Zoe.
+
+Let's check that our offer was successful:
+
+<<< @/snippets/test-intro-zoe.js#offerResult
+
+The AtomicSwap contract gives back this particular message for our
+offer: "The offer has been accepted. Once the contract has been
+completed, please check your payout." Other contracts and offers may
+return something different. The `offerResult` is entirely up to the
+contract.
+
+### Getting the payouts
+
+Because this was the atomic swap contract, the contract is over once
+the second party escrows the correct assets. Let's get our payout of
+moola using the Keyword we used ('Asset'):
+
+<<< @/snippets/test-intro-zoe.js#getPayout
+
+Alice receives her payouts too:
+
+<<< @/snippets/test-intro-zoe.js#alicePayout
+
+## Writing and installing a contract
+
+We've gone over the process of inspecting an invitation, making an
+offer, receiving a `UserSeat` and getting a payout, but how did Alice
+create the contract instance in the first place?
+
+Let's pretend that Alice wrote the contract from scratch, even
+AtomicSwap is already written as one of our example contracts ([see
+the full AtomicSwap code
+here](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/atomicSwap.js)).
+Note: All Zoe contracts must have a certain format, shown below.
+
+::: details Show contract format
+<<< @/snippets/contract-format.js
+:::
+
+
+
+Let's imagine that Alice fills in this code with the particulars of
+the [AtomicSwap implementation](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/atomicSwap.js). 
+
+To install this particular code, Alice must first bundle it off-chain, meaning
+that the code and its imports are flattened together:
+
+<<< @/snippets/test-intro-zoe.js#importBundleSource
```

Comment:
This is empty. Was something supposed to go here?

Reply:
Oh wow, I think `yarn lint-fix` really messed it up. I checked this before linting, but not after. Thanks, I'll fix.

Reply:
Fixed